### DISCUSSION TITLE
Welcome to ytDownloader Discussions!

### DISCUSSION THREAD

**Discussion by aandrew-me** [aandrew-me/ytDownloader#32] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by charleslana** [comment]


**Reply by aandrew-me** [reply]
Hey!
Here is my discord username - Andrэw#0307

**Comment by dustinrodriguez** [comment]
Hello, I was wondering about what is the difference between downloading audio and extracting audio?
Thanks for your time.

**Comment by aandrew-me** [comment]
Not all platforms provide audio separately. In that case audio can be Extracted from video

**Comment by ChadGarion25** [comment]
I was learning to Linux on an old tablet when I saw this  application on the Software Manger and found out I could install it on Windows too. I'm eager to ditch the online tool I've been using up to this point so I could avoid trackers.

I was wondering if there is anywhere to configure MP3 tags specifically for downloads? The previous downloader I was using put the Video title as the Track Title Tag and Album Title Tag and put the YouTube Channel as the Artist Tag. Your downloader does make an automatic thumbnail (which is good because I was having trouble making that work on my own) and I could use a Python Script to fill in the other Tags from the YouTube ID at the end of file name doing a request web scrape. However, I saw in the preferences an option to use a configuration file and wanted to double check if there was some advanced functionality to do this within the app?

**Edit -** Just to be clear, I'm not a coder; I'm just trying to learn some things I always wanted to by doing little side projects and automation

**Reply by aandrew-me** [reply]
I will check the tag part.
Regarding the configuration file, you can put configuration options from yt-dlp - https://github.com/yt-dlp/yt-dlp?tab=readme-ov-file#usage-and-options

**Comment by hackercpdd** [comment]
我电脑为什么不支持

**Reply by hackercpdd** [reply]
<img width="979" height="708" alt="d962f0f93f90d2b9fd33137b13655b8b" src="https://github.com/user-attachments/assets/cd2ece95-7780-42a2-90da-e1e0328cc6f9" />

**Reply by aandrew-me** [reply]
Provide info about your computer (operating system, processor info etc.)

**Reply by hackercpdd** [reply]
> 请提供您的电脑信息（操作系统、处理器信息等）。

![Uploading 68b2d92c7c4aa76b686db395422ae927.png…]()

**Reply by hackercpdd** [reply]
> 请提供您的电脑信息（操作系统、处理器信息等）。

win10

**Comment by hackercpdd** [comment]
<img width="886" height="155" alt="68b2d92c7c4aa76b686db395422ae927" src="https://github.com/user-attachments/assets/076d9792-9973-487f-b61c-78df1a97bc5e" />

**Reply by hackercpdd** [reply]
就这个

**Reply by aandrew-me** [reply]
Okay that's very strange. I have never seen someone facing this issue before.

**Reply by hackercpdd** [reply]
> 好吧，这很奇怪。我以前从未见过有人遇到过这种情况。

好难受

**Reply by hackercpdd** [reply]
> > 好吧，这很奇怪。我以前从未见过有人遇到过这种情况。
> 
> 好难受

需要翻墙才能用吗还是正常就可以

**Reply by hackercpdd** [reply]
> > > 好吧，这很奇怪。我以前从未见过有人遇到过这种情况。
> > 
> > 
> > 好难受
> 
> 需要翻墙才能用吗还是正常就可以

第一次打开就是我的电脑无法用，然后在进去就一直显示下载必要插件

**Reply by aandrew-me** [reply]
So it worked, it downloaded the necessary files. I don't know why Windows is showing that message.

**Reply by hackercpdd** [reply]
> 它成功了，必要的文件都下载好了。我不知道为什么Windows会显示那个消息。

<img width="988" height="738" alt="4772045b0cea0225bdc48554a6899563" src="https://github.com/user-attachments/assets/66eabfd7-4b0a-4008-9252-aa4833af2dc7" />
一直显示这个

**Reply by aandrew-me** [reply]
Does the download not finish?

**Reply by hackercpdd** [reply]
> 下载无法完成吗？

一直卡在这里，在点击就是回到此应用无法在我电脑上运行

**Reply by aandrew-me** [reply]
I genuinely don't know then. :(

**Reply by huihaishi02-ops** [reply]
Why does it keep showing “Processing”?
![Uploading ScreenShot_2025-11-10_140543_526.png…]()

**Comment by Klim-GG** [comment]
AMD Ryzen 7 5700X / 16,0 ГБ / NVIDIA GeForce GTX 1660 SUPER
Windows 11 Pro 25H2

<img width="1264" height="952" alt="2025-12-29 02 30 08" src="https://github.com/user-attachments/assets/d926f679-3ae3-4496-b91d-9e9b65f373b6" />

**Reply by Vitalicus** [reply]
sudo snap saved
sudo snap forget snapshot-id

and reinstall

**Comment by TASTYn** [comment]
Why are some videos not available for download in resolutions higher than 1080p?
I wanted to download videos in 4k resolution, however, I can't see above 1012p, I may be stupid) 
I'd be glad if you could help)
![photo_2026-03-09_20-09-57](https://github.com/user-attachments/assets/ebe095ce-f40f-4a93-b910-933a9b7abe43)

**Reply by aandrew-me** [reply]
Enable this

<img width="1822" height="194" alt="image" src="https://github.com/user-attachments/assets/29ed0874-f054-409a-8ab4-87e0117a9563" />

**Reply by TASTYn** [reply]
I checked, unfortunately it doesn't work(
<img width="1020" height="669" alt="{53F29F4C-CE5C-437A-95EC-A72F9A7CBD5A}" src="https://github.com/user-attachments/assets/14686a98-688e-426f-96c3-b96f04b52e48" />

**Reply by aandrew-me** [reply]
Paste the video link here

**Reply by TASTYn** [reply]
https://www.youtube.com/watch?v=dwol_f55rFs&list=PL0f8mw3wmmQyyYCISJ08Q6F-YBMHRJ4Ua

**Reply by aandrew-me** [reply]
Here -
<img width="2638" height="1574" alt="image" src="https://github.com/user-attachments/assets/731f9f95-cc1a-4939-91aa-d1f239bbb9d2" />

**Comment by benboerba** [comment]
”sudo xattr -r -d com.apple.quarantine /Applications/YTDownloader.app
Password:
xattr: No such file: /Applications/YTDownloader.app   “   May I ask what the issue is with this prompt?   macos

**Reply by aandrew-me** [reply]
Did you move the app file to Applications?
